### ISSUE TITLE
Future Features

### ISSUE THREAD

**Issue by Harleyoc1** [Harleyoc1/DynamicTreesNaturesAura#8]
- [ ] Better integration with the decaying feature. Currently dynamic leaves can decay, however the leaves on a dynamic tree grow back and so decaying has less of an impact. 
- [ ] Dynamic ritual of the forest. The regular ritual of the forest, but using a dynamic tree instead of a vanilla one.

**Comment by joe-vettek** [comment]
Is the second implemented? I plan to use dynamic trees in One Block to build a more difficult wood acquisition environment, but it seems that it hasn't been implemented yet. It looks like I have to cancel the replacement vanilla sapling configuration.

**Comment by joe-vettek** [comment]
I have a suggestion that only when the saplings put into the forest ritual will not become a dynamic tree, I think it only needs to check the multi-block structure.

**Comment by Harleyoc1** [comment]
No, neither are implemented. It's a checklist I will get around to eventually. I plan on looking at the second with a fresh set of eyes very soon. This will work like the regular ritual of the forest except you plant a dynamic sapling and once the central log grows to a specific thickness the tree will be considered "grown" like a vanilla tree and the ritual will begin like it does in the regular ritual.
